datasetinsights/constants.py

```diff
@@ -53,3 +53,6 @@
     "allow_extra_args": True,
 }
 DEFAULT_DATASET_VERSION = "latest"
+
+MLFLOW_TRACKER = "mlflow"
+DEFAULT_RUN_NAME = "run_" + str(datetime.now())
```

Comment:
I think we should take this out of this global constants file. This is fairly specific to mlflow anyways, so put this in the tracker factory or the mlflow tracker itself.

Also, a better strategy to generate ids that benefit from being human readable is something like `202011121633-foo-bar` where the first token is just a readable datetime and the second a random adjective and random object which makes the id much easier to share, find and remember.

Reply:
Changed with this kind of name: train_20201113-162123, eval_20201113-162125

Reply:
why not use dash everywhere? instead of mixing `-` and `_`?